wave_vision/src/matcher/brute_force_matcher.cpp

```diff
@@ -0,0 +1,80 @@
+// Libwave Headers
+#include "wave/vision/matcher/brute_force_matcher.hpp"
+
+namespace wave {
+
+// Default constructor. Struct may be default or user defined.
+BruteForceMatcher::BruteForceMatcher(const BFMatcherParams &config) {
+    // Ensure parameters are valid
+    this->checkConfiguration(config);
+
+    // Create cv::BFMatcher object with the desired parameters
+    this->brute_force_matcher =
+      cv::BFMatcher::create(config.norm_type, config.cross_check);
+
+    // Store configuration parameters within member struct
+    this->current_config = config;
+}
+
+BruteForceMatcher::BruteForceMatcher(const std::string &config_path) {
```

Comment:
Or, @leokoppel do you mind if we just push this as is, and then I fix them for all of the modules at once? This way master will have the same implementation at each time.